### PR TITLE
Health care questionnaire semantic markup

### PR DESCRIPTION
**Pull request by mdewey**
## Description
- Updated the semantic mark up for todo and completed cards
- removed nav from tabs
- Updated the boldness of labels
- Added role for mark up

## Testing done
- unit

## Acceptance criteria
- [ ] matches recommendations from accessibility

## Definition of done
- [To Card Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24589)
- [Nav Removal](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24582)

### CODE REVIEW

**Review by stephenBarrs — APPROVED** [review]

File: src/applications/health-care-questionnaire/list/questionnaire-list/components/CompletedQuestionnaires/index.jsx [hunk 1]
```diff
@@ -27,9 +27,11 @@ const index = props => {
               }
             />
           ) : (
+            // eslint-disable-next-line jsx-a11y/no-redundant-roles
```
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/health-care-questionnaire/list/questionnaire-list/components/CompletedQuestionnaires/index.jsx:30]: This was advised by the VSP accessibility specialist to ensure Safari + VO announces list items properly.

File: src/applications/health-care-questionnaire/list/questionnaire-list/components/ToDoQuestionnaires/index.jsx [hunk 2]
```diff
@@ -20,10 +20,11 @@ const Index = props => {
               }
             />
           ) : (
+            // eslint-disable-next-line jsx-a11y/no-redundant-roles
```
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/health-care-questionnaire/list/questionnaire-list/components/ToDoQuestionnaires/index.jsx:23]: This was advised by the VSP accessibility specialist to ensure Safari + VO announces list items properly.

File: src/applications/health-care-questionnaire/list/questionnaire-list/components/CompletedQuestionnaires/index.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/health-care-questionnaire/list/questionnaire-list/components/CompletedQuestionnaires/index.jsx:30]: This was advised by the VSP accessibility specialist to ensure Safari + VO announces list items properly.

File: src/applications/health-care-questionnaire/list/questionnaire-list/components/QuestionnaireItem/index.jsx [hunk 3]
```diff
@@ -24,16 +24,20 @@ const index = props => {
...
+      <dl className="vads-u-margin-bottom--0p5">
+        <dt data-testid="appointment-status">
+          For your {isCancelled ? 'canceled or rescheduled ' : ''}
+          appointment at
+        </dt>
+        <dd data-testid="appointment-location">
+          {clinicName}, {facilityName}
+          {extraText && `. ${extraText}`}
+        </dd>
+        {DueDate && <DueDate />}
```
Comment (1Copenut) [inline src/applications/health-care-questionnaire/list/questionnaire-list/components/QuestionnaireItem/index.jsx:39]: Let's keep an eye on this in rendered HTML Axe is particular about anything in a DL that's not contained in a DIV, DT, or DD.

File: src/applications/health-care-questionnaire/list/questionnaire-list/components/ToDoQuestionnaires/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (1Copenut) [reply src/applications/health-care-questionnaire/list/questionnaire-list/components/ToDoQuestionnaires/index.jsx:23]: This was advised by the VSP accessibility specialist to ensure Safari + VO announces list items properly.

**Review by 1Copenut — APPROVED** [review]

**Review by jhonny218 — APPROVED** [review]

**Pull request merged**